addons/website_sale_slides/static/src/xml/website_sale_slides_quiz.xml

```diff
@@ -2,29 +2,34 @@
 <templates xml:space="preserve">
     <t t-extend="slide.slide.quiz.validation">
         <t t-jquery="#validation" t-operation="append">
-            <div t-if="widget.readonly &amp;&amp; widget.publicUser &amp;&amp; widget.channel.channelEnroll == 'payment'" class="alert alert-info d-flex align-items-center justify-content-between">
+            <div t-if="!widget.isMember &amp;&amp; widget.publicUser &amp;&amp; widget.channel.channelEnroll == 'payment'" class="alert alert-info d-flex align-items-center justify-content-between">
                 <div>
-                    <b class="h5 mb-0">Sign in and buy the course to join it and take the quiz!</b>
+                    <b class="h5 mb-0">Sign in and buy the course to take the quiz!</b>
                     <span class="my-0 h4" style="line-height: 1">
                         <span title="Succeed and gain karma" aria-label="Succeed and gain karma" class="badge badge-pill badge-warning text-white font-weight-bold ml-3 px-2 py-1">
-                            + <t t-esc="widget.quiz.quizKarmaGain"/> XP
+                    + <t t-esc="widget.quiz.quizKarmaGain"/> XP
```

Comment:
Why did you change the indentation here?

Reply:
no reason, removed it